vm/src/stdlib/time.rs

```diff
@@ -91,15 +91,19 @@ mod time {
 
     #[cfg(all(target_arch = "wasm32", not(target_os = "wasi")))]
     fn _time(_vm: &VirtualMachine) -> PyResult<f64> {
-        use wasm_bindgen::prelude::*;
-        #[wasm_bindgen]
-        extern "C" {
-            type Date;
-            #[wasm_bindgen(static_method_of = Date)]
-            fn now() -> f64;
+        #[cfg(feature = "wasmbind")]
+        {
+            use wasm_bindgen::prelude::*;
+            #[wasm_bindgen]
+            extern "C" {
+                type Date;
+                #[wasm_bindgen(static_method_of = Date)]
+                fn now() -> f64;
+            }
+            // Date.now returns unix time in milliseconds, we want it in seconds
+            return Ok(Date::now() / 1000.0);
         }
-        // Date.now returns unix time in milliseconds, we want it in seconds
-        Ok(Date::now() / 1000.0)
+        panic!{"No date available"}
     }
```

Comment:
This should be
```rust
#[cfg(all(target_arch = "wasm32", not(target_os = "wasi"), feature = "wasmbind"))]
fn _time(_vm: &VirtualMachine) -> PyResult<f64> {
    // ..
}
```

Reply:
Addressed in https://github.com/RustPython/RustPython/pull/4875